paper/src/main/java/com/denizenscript/denizen/paper/events/PlayerCompletesAdvancementScriptEventPaperImpl.java

```diff
@@ -0,0 +1,60 @@
+package com.denizenscript.denizen.paper.events;
+
+import com.denizenscript.denizen.events.player.PlayerCompletesAdvancementScriptEvent;
+import com.denizenscript.denizen.paper.PaperModule;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import net.kyori.adventure.text.Component;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.player.PlayerAdvancementDoneEvent;
+
+public class PlayerCompletesAdvancementScriptEventPaperImpl extends PlayerCompletesAdvancementScriptEvent {
+
+    public PlayerCompletesAdvancementScriptEventPaperImpl() {
+        instance = this;
+    }
+
+    @Override
+    public ObjectTag getContext(String name) {
+        if (name.equals("criteria")) {

```

Comment:
No need to handle the `criteria` and `advancement` contexts here, as they are already handled in the super method